### ISSUE TITLE
Unhandled Exception: MissingPluginException(No implementation found for method share on channel dev.fluttercommunity.plus/share)

### ISSUE THREAD

**Issue by unilogica** [fluttercommunity/plus_plugins#1380]
### What is your question?

I'm trying example from Readme:

 Share.share('check out my website https://example.com');

```
[VERBOSE-2:dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method share on channel dev.fluttercommunity.plus/share)
#0      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:294
<asynchronous suspension>
```

Using last version from share_plus. I need to implement something thats not documented?

### Checklist before submitting a question

- [X] I Google'd a solution and I couldn't find it
- [X] I searched on StackOverflow for a solution and I couldn't find it
- [X] I read the README.md file of the plugin
- [X] I am using the latest version of the plugin
- [X] All dependencies are up to date with `flutter pub upgrade`
- [X] I did a `flutter clean`
- [X] I tried running the example project

**Comment by riccardocescon** [comment]
@unilogica how did you solved this?

**Comment by riccardocescon** [comment]
For anyone having this issue: just stop and restart the app

**Comment by unilogica** [comment]
I was reading the documentation wrong. My case was resolved when I updated flutter, cleared the cache and reinstalled the project from scratch with the right dependencies.

Then another problem I had was not generating a temporary file in the right place to share (I was duplicating the file in the gallery), but this was also described in the documentation. Sometimes a tight deadline makes us not read what is written even in the spotlight.
